pyvista/utilities/cells.py

```diff
@@ -0,0 +1,77 @@
+"""pyvista wrapping of vtkCellArray."""
+import numpy as np
+from vtk.util.numpy_support import numpy_to_vtkIdTypeArray, vtk_to_numpy
+from vtk import vtkCellArray
+
+import pyvista
+
+
+def numpy_to_idarr(ind, deep=False, return_ind=False):
+    """Safely convert a numpy array to a vtkIdTypeArray."""
+    try:
+        ind = np.asarray(ind)
+    except:
+        raise TypeError('Indices must be either a mask, array, list, or iterable')
+
+    if ind.dtype == np.bool:
+        ind = ind.nonzero()[0].astype(pyvista.ID_TYPE)
+    elif ind.dtype != pyvista.ID_TYPE:
+        ind = ind.astype(pyvista.ID_TYPE)
+    elif not ind.flags['C_CONTIGUOUS']:
+        ind = np.ascontiguousarray(ind, dtype=pyvista.ID_TYPE)
+
+    # must ravel or segfault when saving MultiBlock
+    vtk_idarr = numpy_to_vtkIdTypeArray(ind.ravel(), deep=deep)
+    if return_ind:
+        return vtk_idarr, ind
+    return vtk_idarr
+
+
+class CellArray(vtkCellArray):
+    """pyvista wrapping of vtkCellArray.
+
+    Provides convenience functions to simplify creating a CellArray from
+    a numpy array or list.
+
+    Import an array of data with the legacy vtkCellArray layout, e.g.
+
+    ``{ n0, p0_0, p0_1, ..., p0_n, n1, p1_0, p1_1, ..., p1_n, ... }``
+    Where n0 is the number of points in cell 0, and pX_Y is the Y'th
+    point in cell X.
+
+    Examples
+    --------
+    Create a cell array containing two triangles.
+    >>> from pyvista.core.cells import CellArray
```

Comment:
```suggestion
    >>> from pyvista.utilities.cells import CellArray
```

Reply:
Thanks!  I usually run `pytest` locally, but I need to add a git hook to run the doc-tests...